main.js

```diff
@@ -3,6 +3,7 @@ var $ = require('jquery');
 $(function(){
 	var $emailContainer = $('.convio-survey');
 	var surveyID = $emailContainer.attr('data-surveyid');
+	var buttonLabel = ($emailContainer.attr('data-button-label') == 'undefined') ? $emailContainer.attr('data-button-label') : 'Submit';
```

Comment:
@noahdklein The logic here is now wrong. It's says that if the attribute data-button-label is undefined, then use the data-button-label attribute as the button label, and if it isn't undefined, use "Submit." You want the opposite, which was why I had it as !=.

It's looking for the data-button-label label on the object with the class "convio-survey," so that's where you need to place the attribute in the html. IE:

`<div class="convio-survey" data-surveyid="14288" data-button-label="Get My Certificate" style="display: none;">`